chain/default_store.go

```diff
@@ -384,11 +390,11 @@ func (store *DefaultStore) BlockHeight() (uint64, error) {
 
 // ActorFromLatestState gets the latest state and retrieves an actor from it.
 func (store *DefaultStore) ActorFromLatestState(ctx context.Context, addr address.Address) (*actor.Actor, error) {
-	tsas, err := store.GetTipSetAndState(store.GetHead())
+	tssr, err := store.GetTipSetStateRoot(store.GetHead())
```

Comment:
maybe should change tssr to something like tsStateRoot?

Reply:
That sounds reasonable, but I don't have any strong opinions either way. @anorth, do you?

Reply:
Yes, `stateRoot` or `stateCid` would be better.